src/libraries/Microsoft.PowerFx.Core/Public/Values/TableValue.cs

```diff
@@ -78,6 +79,16 @@ private static ErrorValue ArgumentOutOfRange(IRContext irContext)
             });
         }
 
+        // Return appended value 
+        // - Error, 
+        // - with updated values
+        // Async because derived classes may back this with a network call. 
+        public virtual async Task<DValue<RecordValue>> AppendAsync(RecordValue record)
+        {
+            // fails by default
```

Comment:
>// fails by default [](http://example.com/codeflow?start=12&length=19)

Will need to confirm the exact error here. Will probably need to be a runtime ErrorValue (DValue allows that) rather than an exception

Reply:
Got it. Added this to my personal notes as reminder.